### ISSUE TITLE
Lists needs two refinements for clarity

### ISSUE THREAD

**Issue by Tasilee** [AtlasOfLivingAustralia/specieslist-webapp#153]
1. We need rollovers on List types at https://lists.ala.org.au as some of the list types will be obscure for the average user (and me).
1. We need to separate the List types into two categories: **List Types** that include   

- Species characters list 
- Conservation list
- Sensitive list of species
- Common habitat of species
- Area checklist 
- Common trait of species 
- Spatial portal defined list 
- Profile list 
- Test list 
- Other list

and **List Tags** that include

- Is private
- Included in species pages
- Authoritative
- Invasive
- Threatened
- Part of the sensitive data service
- Region provided

**Comment by Tasilee** [comment]
These are the definitions as best I can figure

LIST TYPES
Area checklist (Species within a formally defined area: 362) 
Common habitat of species (Species linked to one habitat type: 18) 
Common trait of species (Species with the same trait/attribute: 44) 
Conservation list (Species with any conservation status: 65) 
List of sensitive species (Species with locations generalized: 49) 
Other (Does not fit current List types: 48) 
Profile list (Lists used by the ALA’s Profile tool:108) 
Region provided (17) 
Spatial portal defined list (List generated on-the-fly in the ALA’s Spatial Portal: 3839) 
Species characters list (Species with an appended set of traits/attributes :249) 
Test list (Temporary list (anticipate deletion after one month): 117) 

TAGS 
Authoritative (Created by acknowledged responsible agency, committee or expert 74) 
Included in species pages (List is referenced on the species summary page: 78) 
Private list (4170) 
Invasive species list (Species with any invasive status in the Australian region: 25) 
Member of Sensitive Data Service list (Species have authoritative sensitivities: 10) 
Private list (Not publicly visible: 4170) 
Region provided (A list-associated area has been defined: 17)  
Threatened species list (Species with any conservation status in Australia: 20)

**Comment by elywallis** [comment]
Hi @Tasilee 
A few questions:
How is this LIST TYPE
Conservation list (Species with any conservation status: 65)
different from this TAG
Threatened species list (Species with any conservation status in Australia: 20)

How does this LIST TYPE
List of sensitive species (Species with locations generalized: 49)
differ from this TAG
Member of Sensitive Data Service list (Species have authoritative sensitivities: 10)

And why is this a TAG not a LIST TYPE?
Invasive species list (Species with any invasive status in the Australian region: 25)

To my eye, these are confusing and it's difficult to see what defines a TAG from a LIST TYPE
my 2 cents

**Comment by Tasilee** [comment]
Put it this way, somehow, I was never advised when new lists were created so now I am trying to 'put the fires out'. In short, there are indeed plenty of anomalies that we need to correct, but as a first step, we need to know rough definitions for rollovers as users had nothing that described the lists. That is unacceptable. 

What I am calling TAGS are different to List types. If you look now, you don't see any discrimination. But Rita is fixing this today I hope. 

Part of this review that I have done first needed to expose what was there and that meant indexing both TYPES and TAGS.

TAGS are added descriptors that are associated with any list type. As List admin, I am usually the one who will attach 'Authoritative', and/or 'Invasive' or 'Threatened'. Probably Miles has added a few. He adds the SDS TAGs. The only TAG a user can apply (I think as I am admin) is 'Is Private' (which too many do, sadly.

There ARE overlaps in TYPES and TAGS due to ad hoc additions I wasn't informed about. To address your specifics

How is this LIST TYPE
Conservation list (Species with any conservation status: 65)
different from this TAG
Threatened species list (Species with any conservation status in Australia: 20)

...the latter is manually added, usually with the TAG 'authoritative'. Point being anything can be put into a list called a 'conservation list' but only admins can add the 'Threatened' TAG. In short, not all 'conservation list' TYPES' will have a 'threatened TAG'. If you look at my lists, I have EPBC migratory species as "List of sensitive species" (NOT!! "Sensitive list of species as was wrongly added - and been corrected hopefully). So, there is more an overlap with list TYPEs sensitive and conservation.

How does this LIST TYPE
List of sensitive species (Species with locations generalized: 49)
differ from this TAG
Member of Sensitive Data Service list (Species have authoritative sensitivities: 10)

...Not all lists of sensitive species are in the SDS but this needs Miles to comment. My definitions are as best as I can figure at the moment but once we have them clarified, we DO need a KB article. Whether this happens before or after rationalization, TBA.

And why is this a TAG not a LIST TYPE?
Invasive species list (Species with any invasive status in the Australian region: 25)

....this is the partner TAG to 'Threatened'. I have added the GRIIS list under as "Common trait of species" (invasive).

In short, what you see here are a series of steps toward improving and correcting.

**Comment by nickdos** [comment]
I left some comments for commit 4ff3d33

**Comment by Rita-C** [comment]
> I left some comments for commit [4ff3d33](https://github.com/AtlasOfLivingAustralia/specieslist-webapp/commit/4ff3d33b4308706f5a4b650b70b3888b2a50bf1c)

thanks, code updated.
